src/navigation/RootRouter.tsx

```diff
@@ -40,6 +18,7 @@ const RootRouter = () => {
         {routes.map((route, i) => {
           return route.protected ? (
```

Comment:
@mastertalent Try to implement Guard which is wrapping the route tag. thanks.
<ProtectGuard>
<Route>
<ProtectGuard>